<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressDialog</class>
 <widget class="QDialog" name="CompressDialog">
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>367</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Image Info</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_compression">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>367</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Compression Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Format:</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="formatComboBox">
           <item>
            <property name="text">
             <string>Uncompressed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC1 (DXT1)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC1a (DXT1a)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC2 (DXT3)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC3 (DXT5)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BC5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="formatOptions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_colorweights">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Color Weights</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter</set>
             </property>
             <layout class="QVBoxLayout" name="_2">
              <property name="topMargin">
               <number>12</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="_3">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Red</string>
                  </property>
                  <property name="buddy">
                   <cstring>redSpinBox</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="redSpinBox">
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="_4">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Green</string>
                  </property>
                  <property name="buddy">
                   <cstring>greenSpinBox</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="greenSpinBox">
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="_5">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Blue</string>
                  </property>
                  <property name="buddy">
                   <cstring>blueSpinBox</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="blueSpinBox">
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.050000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="_6">
                <item>
                 <widget class="QToolButton" name="uniformButton">
                  <property name="text">
                   <string>Uniform</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="luminanceButton">
                  <property name="text">
                   <string>Luminance</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_pixelformat">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Pixel Format:</string>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="pixelformatComboBox">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>Custom</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>R8G8B8A8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>R5G6B5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>A1</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Color Type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Fixed</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Float</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Red Bits:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="redbitSpinBox">
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Green Bits:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="greenbitSpinBox">
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Blue Bits:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="bluebitSpinBox">
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Alpha Bits:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="alphabitSpinBox">
               <property name="maximum">
                <number>32</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ditherColorCheckBox">
         <property name="text">
          <string>Dither Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Dither Alpha</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_resize">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>367</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Resize Options</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_mipmap">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>357</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Mipmap Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="generateMipmapsCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Generate mipmaps</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_8">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="mipmapFilterLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Filter:</string>
           </property>
           <property name="buddy">
            <cstring>mipmapFilterComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mipmapFilterComboBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kaiser</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mipmapFilterSettings">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_9">
         <item>
          <widget class="QCheckBox" name="limitMipmapsCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Limit Mipmaps</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="maxLevelLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Max Level:</string>
           </property>
           <property name="buddy">
            <cstring>maxLevelSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxLevelSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CompressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CompressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
